src/redux/data.ts

```diff
@@ -29,6 +29,11 @@ export interface Address {
 
 export interface PrimaryPerson extends Person {
   address: Address
+  isTaxpayerDependent: boolean
+}
+
+export interface Spouse extends Person {
```

Comment:
This is a new interface. Check out the end of redux/validate.ts. 

I have found that is necessary to resolve the error you hit.